.claude/commands/commit.md

```
@@ -0,0 +1,57 @@
+---
+description: Create a git commit following project standards
+argument-hint: [optional-commit-description]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)